soepy/python/solve/solve_auxiliary.py

```diff
@@ -169,6 +169,9 @@ def pyth_backward_induction(attr_dict, state_space_args):
 
     draws_emax = draw_disturbances((num_periods, num_draws_emax), shocks_cov, seed_emax)
 
+    # Construct covariates
+    covariates = construct_covariates(states_all, states_number_period,  max_states_period, attr_dict)
```

Comment:
It seems like the quality test is failing since your code isn't inline with our standards. For instance this line has more than 90 characters. Running the black formatter on the module should solve the problem. Did you deactivate the precommit hooks?